datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTab.tsx

```diff
@@ -87,6 +158,8 @@ export const SchemaTab = ({ properties }: { properties?: any }) => {
                             editMode={editMode}
                             editableSchemaMetadata={editableSchemaMetadata}
                             usageStats={usageStats}
+                            getSchemaBlameResult={getSchemaBlameData?.getSchemaBlame}
```

Comment:
minor nit: you can call this "schemaBlame"

Reply:
Agreed